### DISCUSSION TITLE
After deploying the code to docker, the styles, opacity & the fonts are not coming up properly

### DISCUSSION THREAD

**Discussion by dreamweavetechnologies** [primefaces/community#720] [PrimeNG Templates] [answered]
Team, 
When i load the verona-ng locally it all works fine. But when i built it using docker and load it, it is not loading the styles, opacity and fonts. Here is the docker file used

**Dockerfile**
# Stage 1: Build the Angular application
FROM node:18.13 as build
WORKDIR /app
COPY package.json package-lock.json ./
RUN npm install
RUN npm install -g @angular/cli
COPY . .
RUN ng build --configuration production

# Stage 2: Serve the application with Nginx
FROM nginx:1.19
COPY --from=build /app/dist/verona-ng /usr/share/nginx/html
COPY ./nginx-custom.conf /etc/nginx/conf.d/default.conf

**nginx-custom.conf**
server {
    listen 9093;
    location / {
        root /usr/share/nginx/html;
        index index.html index.htm;
        try_files $uri $uri/ /index.html =404;
    }
}

The UI is showing like this

![image](https://github.com/primefaces/community/assets/144503896/eab9bedb-39d3-4df9-9bfb-20c858dc66de)
![image](https://github.com/primefaces/community/assets/144503896/6eb87bcb-09d0-473d-b5b0-9c4e81c590e9)

Can anyone please help

**Comment by cetincakiroglu** [comment]
Hi,

Have you tried `sass --watch src/assets:src/assets --no-source-map` ?

**Reply by dreamweavetechnologies** [reply]
Thanks for your reponse, i think for some reason scss files in my assets folders are not getting converted into css. 
Am seeing this error in the docker build
#13 [build 7/7] RUN ng build --configuration production
#13 2.364 - Generating browser application bundles (phase: setup)...
#13 85.11 ✔ Browser application bundle generation complete.
#13 89.15 ✔ Browser application bundle generation complete.
#13 89.15 - Copying assets...
#13 89.32 ✔ Copying assets complete.
#13 89.32 - Generating index html...
#13 89.67 - Generating index html...
#13 89.68 Unable to locate stylesheet: /app/dist/verona-ng/assets/layout/styles/layout/preloading.css
#13 89.68 Unable to locate stylesheet: /app/dist/verona-ng/assets/layout/styles/theme/theme-light/indigo/theme.css


Where should i try adding this command - sass --watch src/assets:src/assets --no-source-map ?

**Reply by dreamweavetechnologies** [reply, marked as answer]
Thank you so much , It worked after i update the docker file like this 

# Stage 1: Build the Angular application
FROM node:18.18.0 as build
WORKDIR /app
COPY package.json package-lock.json ./
RUN npm install -g @angular/cli
RUN npm install -g sass
RUN npm install
COPY . .
RUN sass src/assets:src/assets --no-source-map
RUN ng build --configuration production

# Stage 2: Serve the application with Nginx
FROM nginx:1.19
COPY --from=build /app/dist/verona-ng /usr/share/nginx/html
COPY ./nginx-custom.conf /etc/nginx/conf.d/default.conf
